reference/conanfile/attributes.rst

```diff
@@ -0,0 +1,1022 @@
+.. spelling::
+
+  ing
+  ver
+
+.. currentmodule:: conan
+
+.. _conan_conanfile_attributes:
+
+Attributes
+==========
+
+
+name
+----
+
+.. autoattribute:: ConanFile.name
+
+
+   A valid name has:
+
+    - A minimum of 2 and a maximum of 101 characters (though shorter names are recommended).
+    - Matches the following regex ``^[a-z0-9_][a-z0-9_+.-]{1,100}$``: so starts with alphanumeric or ``_``,
+      then from 1 to 100 characters between alphanumeric, ``_``, ``+``, ``.`` or ``-``.
+
+   The name is only necessary for ``export``-ing the recipe into the local cache (``export`` and ``create`` commands),
+   if they are not defined in the command line.
+
```

Comment:
Are all of the indents on purpouse? Shouldn't it be aligned to the left?
<img width="717" alt="image" src="https://user-images.githubusercontent.com/5045666/170238924-5523482f-f663-4bf4-b524-44d97681b6dd.png">

Reply:
It was on purpose, so the text outside the ``docstrings is in the same tabulation, so you don't care if that is from docstring or from the docs. But could be changed.

Reply:
I prefer to have them differentiated, but let's discuss and see what others think.

Reply:
Everything is now automatically without margin left.